cartography/stats.py

```diff
@@ -97,7 +97,9 @@ def set_stats_client(stats_client: StatsClient) -> None:
     """
     This is used to set the module level stats client configured to talk with a statsd host
     """
-    global _scoped_stats_client
+    # global _scoped_stats_client
+    # Above line is throwing below error, so commenting the code for now
+    # [F824] `global _scoped_stats_client` is unused: name is never assigned in scope
```

Comment:
```suggestion
    global _scoped_stats_client  # noqa: F824
```
This error is a bit misleading - the linter is expecting some sort of direct assignment, but instead we're defining a global object and calling a method on it, which is perfectly valid Python! Let's bypass the linter error instead